src/api/app/apis/gitea_api/v1/client.rb

```diff
@@ -0,0 +1,69 @@
+module GiteaAPI
+  module V1
+    class Client
+      HTTP_OK_CODE = 200
+      HTTP_CREATED_CODE = 201
+      HTTP_BAD_REQUEST_CODE = 400
+      HTTP_UNAUTHORIZED_CODE = 401
+      HTTP_FORBIDDEN_CODE = 403
+      HTTP_NOT_FOUND_CODE = 404
+
+      GiteaApiError = Class.new(StandardError)
+      BadRequestError = Class.new(GiteaApiError)
+      UnauthorizedError = Class.new(GiteaApiError)
+      ForbiddenError = Class.new(GiteaApiError)
+      NotFoundError = Class.new(GiteaApiError)
+      ApiError = Class.new(GiteaApiError)
+
+      def initialize(api_endpoint:, token:)
+        @api_endpoint = api_endpoint + 'api/v1/'
+        @token = token
+      end
+
+      # owner: owner of the repository
+      # repo: name of the repository
+      # sha: sha of the commit
+      # https://try.gitea.io/api/swagger#/repository/repoCreateStatus
+      def create_commit_status(owner:, repo:, sha:, state:, **kwargs)
```

Comment:
```suggestion
      def create_commit_status(owner:, repo_name:, sha:, state:, **kwargs)
```

Reply:
I wanted to stick with the naming used in the api docs of gitea. does this makes sense?

Reply:
Hmm well, okay